lib/console_web/plug/verify_access_token.ex

```diff
@@ -47,28 +47,40 @@ defmodule ConsoleWeb.Plug.VerifyAccessToken do
   def init(default), do: default
 
   def call(conn, _default) do
-    token = conn
-      |> get_req_header("authorization")
-      |> List.first()
-      |> String.replace("Bearer ", "")
-
-    case @access_token_decoder.decode_conn_access_token(token) do
-      %{user_id: user_id, email: email, auth0_id: auth0_id} ->
-        conn
-          |> assign(:user_id, user_id)
-          |> assign(:email, email)
-          |> assign(:auth0_id, auth0_id)
-      :error ->
+    case conn |> get_req_header("authorization") do
+      nil ->
         conn
-        |> send_resp(
-          :forbidden,
-          Poison.encode!(%{
-            type: "forbidden",
-            errors: ["Could not validate your credentials"]
-          })
-        )
-        |> halt()
-    end
+          |> send_resp(
+            :forbidden,
+            Poison.encode!(%{
+              type: "forbidden",
+              errors: ["Authorization header is missing"]
+            })
+          )
+          |> halt()
+      _ ->
+        token = conn
+        |> get_req_header("authorization")
+        |> List.first()
+        |> String.replace("Bearer ", "")
 
+      case @access_token_decoder.decode_conn_access_token(token) do
```

Comment:
we aren't missing an end right?

Reply:
no sir 
![image](https://user-images.githubusercontent.com/51131939/123486863-b9f24980-d5c1-11eb-8c1c-075cbcd08034.png)

Reply:
LOL thanks for the clear highlighting!

Reply:
lol my indent is off so thanks for catching that